routers/private/serv.go

```diff
@@ -61,6 +61,12 @@ func ServNoCommand(ctx *macaron.Context) {
 			})
 			return
 		}
+		if !user.IsActive {
+			ctx.JSON(http.StatusForbidden, map[string]interface{}{
+				"err": "Your account is disabled.",
+			})
+			return
+		}
```

Comment:
Why did you choose HTTP 403 error for this purpose?

Reply:
Which status code do you think is better?

Reply:
403 seems pretty fitting to me.